rules/linux-kernel.dep

```diff
@@ -1,7 +1,6 @@
 
 SPATH       := $($(LINUX_HEADERS_COMMON)_SRC_PATH)
-DEP_FILES   := $(SONIC_COMMON_FILES_LIST) rules/linux-kernel.mk rules/linux-kernel.dep
-DEP_FILES   += $(SONIC_COMMON_BASE_FILES_LIST)
+DEP_FILES   := rules/linux-kernel.mk rules/linux-kernel.dep
```

Comment:
>DEP_FILES [](http://example.com/codeflow?start=0&length=9)

Do you exclude `.arch` file? Should we include it?

Reply:
@qiluo-msft , the file .arch not in the SONIC_COMMON_FILES_LIST or SONIC_COMMON_BASE_FILES_LIST, see https://github.com/Azure/sonic-buildimage/blob/3e4658231421e9f23996ab7699acbc0fa3b9da2b/Makefile.cache#L69 and https://github.com/Azure/sonic-buildimage/blob/3e4658231421e9f23996ab7699acbc0fa3b9da2b/Makefile.cache#L76

We have another option SONIC_COMMON_FLAGS_LIST contains the arch info, not changed.

Reply:
I am wondering if there will be a problem if one CONFIGURED_PLATFORM has two arch. The kernel build depends arch [https://github.com/Azure/sonic-linux-kernel/blob/10b1d7bec7c8336aa0ddbe4c755fd46e8b64b3e0/Makefile#L110](https://github.com/Azure/sonic-linux-kernel/blob/10b1d7bec7c8336aa0ddbe4c755fd46e8b64b3e0/Makefile#L110)

Reply:
Currently, the CONFIGURED_PLATFORM contains arch, such as marvel_armhf, centec-arm64, do we have any other scenarios that have none unique platform name?
To solve the issue, we should add the variable CONFIGURED_ARCH in SONIC_COMMON_FLAGS_LIST, no SONIC_COMMON_FILES_LIST.